scss/ckeditor/skins/tao/scss/dialog.scss

```diff
@@ -0,0 +1,798 @@
+/*

```

Comment:
is the ckeditor skin directly linked somewhere in this repo  ?

Reply:
Yes, it is in `inc/_toolbar.scss`. I was not sure I could remove it.

Reply:
let me check how we can get rid of this

Reply:
we need to keep them for now, since this would require to move inc/_toolbar.scss into 2 places :/